### ISSUE TITLE
Inconsistent documentation for "can_get_message_thread"

### ISSUE THREAD

**Issue by zevlg** [tdlib/td#1263]
Consider next thread in a chat (using TDLib 1.6.9):
```
MSG1 (initiate the thread)
MSG2-1 (as reply to MSG1)
MSG3-2-1 (as reply to MSG2-1)
```

All of the messages will have `can_get_message_thread` property set to `true`, so according to the documentation (@can_get_message_thread True, if the message thread info is available) thread info is available for all the messages, however `getMessageThread` returns "PEER_ID_INVALID" error for MSG2-1 and MSG3-2-1.  Those messages will have `message_thread_id` property set, however there is no method to get `MessageThreadInfo` by thread_id

Should MSG2-1 and MSG3-2-1 have `can_get_message_thread` set to false or `getMessageThread` do not fail on these messages?  Or maybe TDLib should provide a way to get `MessageThreadInfo` directly by thread_id taken from `message_thread_id` property of a message?

Thanks

**Comment by levlam** [comment]
You seems to try to get information about a message thread in a supergroup, which isn't linked to a channel. The ability to get message threads in such supergroups has been enabled yesterday, but in such supergroups message threads can be received only from root messages. TDLib 1.6.9 had no chance to know about such behavior and returns `can_get_message_thread == true` for such messages, which was correct for discussion groups. So, the `can_get_message_thread` flag should be false.

**Comment by zevlg** [comment]
> You seems to try to get information about a message thread in a supergroup, which isn't linked to a channel. The ability to get message threads in such supergroups has been enabled yesterday, but in such supergroups message threads can be received only from root messages. TDLib 1.6.9 had no chance to know about such behavior and returns `can_get_message_thread == true` for such messages, which was correct for discussion groups. So, the `can_get_message_thread` flag should be false.

Noted!  Yeah, I'm talking about ordinary supergroup not linked to a channel.

I have another related question.  `getMessageThreadHistory` accepts `message_id` that starts a thread, so client needs to find that message in the chat to get a thread history, however thread might be too long to calculate first message in a thread.  On the other hand each message in a thread has `message_thread_id` that could be potentially used instead of `message_id` in the `getMessageThreadHistory` method

What do you think about substituting `message_id` to `message_thread_id` in the `getMessageThreadHistory` method?  So client could fetch a thread without traversing reply_to messages to hit the first message in a thread

Thanks!

**Comment by levlam** [comment]
This is not intended to work, despite currently `message_thread_id` and the ID of the top thread message coincide. In official clients there will be no way to open message threads from internal thread messages.

**Comment by levlam** [comment]
This was fixed in TDLib 1.6.10.
